src/selectors/fridge.js

```diff
@@ -106,3 +106,21 @@ export const selectBreaches = createSelector(
     return adjustedBreaches;
   }
 );
+
+export const selectTimestampFormatter = createSelector(
+  [selectTemperatureLogsFromDate, selectTemperatureLogsToDate],
+  (fromDate, toDate) => {
+    const durationInDays = moment(toDate).diff(moment(fromDate), 'days', true);
+
+    const justTime = 'HH:MM';
+    const dateAndTime = 'HH:MM - DD/MM';
+    const justDate = 'DD/MM';
```

Comment:
```suggestion
    const time = 'HH:MM';
    const date = 'DD/MM';
    const dateTime = `${time} - ${date}`;
```

Reply:
Nice - wanted to commit the suggestion but it wasn't quite complete 🤣 


4d6005f